source/fundamentals/auth.txt

```diff
@@ -0,0 +1,384 @@
+=========================
+Authentication Mechanisms
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+In this guide, we show you how to authenticate with MongoDB using each
+**authentication mechanism** available in the MongoDB Community Edition.
+Authentication mechanisms are processes by which the driver and server
+confirm identity and establish trust to ensure security.
+
+The mechanisms that you can use with the latest version of MongoDB Community
+Edition are as follows:
+
+* :ref:`Default <default-auth-mechanism>`
+* :ref:`SCRAM-SHA-256 <scram-sha-256-auth-mechanism>`
+* :ref:`SCRAM-SHA-1 <scram-sha-1-auth-mechanism>`
+* :ref:`MONGODB-CR <mongodb-cr-auth-mechanism>`
+* :ref:`MONGODB-AWS <mongodb-aws-auth-mechanism>`
+* :ref:`X.509 <x509-auth-mechanism>`
+
+If you need to authentication using ``Kerberos`` or ``LDAP`` see the
+:docs:`Enterprise Authentication Mechanisms guide <fundamentals/enterprise-auth>`.
+For more information on establishing a connection to your MongoDB cluster,
+read our :doc:`Connection Guide </fundamentals/connection>`.
+
+Specify an Authentication Mechanism
+-----------------------------------
+
+You can specify your authentication mechanism and credentials when connecting
+to MongoDB using either of the following:
+
+- a connection string
+- a ``MongoCredential`` factory method
+
+A **connection string** or **connection uri** provides instructions on how
+to connect and authenticate to your MongoDB cluster. To specify your
+authentication mechanism using a connection string, include your settings in
+your connection string and pass it to the ``MongoClients.create()`` method
+to instantiate your ``MongoClient``. For descriptions of how to use a
+connection string to specify your authentication method, navigate to the
+relevant mechanism section and select the :guilabel:`Connection String` tab.
+
+The ``MongoCredential`` class contains static factory methods that allow you
+to specify your authentication mechanism and credentials. You can use this
+helper class to keep your code organized. If you choose to use the
+``MongoCredential`` helper class, you need to use the
+``MongoClientSettings.Builder`` class to configure your connection settings
+when constructing your ``MongoClient``. For descriptions of how to use the
+``MongoCredential`` class to specify your authentication method, navigate
+to the relevant mechanism section and select the :guilabel:`MongoCredential`
+tab.
+
+For more information on the aforementioned classes and methods, refer to the
+following API documentation:
+
+- :java-sync-api:`MongoClients.create() </com/mongodb/client/MongoClients.html#create(com.mongodb.ConnectionString)>`
+- :java-sync-api:`MongoClient </com/mongodb/client/MongoClient.html>`
```

Comment:
Is MongoClient relevant here?

Reply:
While it's not tied to the authentication classes/methods, I thought it belonged here because it is the end product of the process of establishing a connection with authentication. I can remove this and the ``MongoClients.create()`` method if you think it's only relevant to include API links to the authentication-specific methods.

Reply:
I don't have strong feelings about it.